e2e-test/trace/src/integrationTest/resources/raw-span-e2e-pipeline-event-type.yml

```diff
@@ -0,0 +1,26 @@
+entry-pipeline:
+  source:
+    otel_trace_source:
+      ssl: false
+      record_type: "event"
+  sink:
+    - pipeline:
+        name: "raw-pipeline"
+raw-pipeline:
+  source:
+    pipeline:
+      name: "entry-pipeline"
+  processor:
+    - otel_trace_raw:
+        root_span_flush_delay: 1 # TODO: remove after 1.1 release
```

Comment:
Apparently this was never removed after the 1.1 release. Maybe we should remove it now.

Reply:
Thanks for the catch!